cmd/kubeadm/app/cmd/config.go

```diff
@@ -47,8 +47,7 @@ import (
 )
 
 var (
-	defaultAPIObjects   = []string{constants.InitConfigurationKind, constants.NodeConfigurationKind}
-	availableAPIObjects = []string{constants.InitConfigurationKind, constants.MasterConfigurationKind, constants.NodeConfigurationKind}
+	defaultAPIObjects = []string{constants.InitConfigurationKind, constants.NodeConfigurationKind}
```

Comment:
i agree with @luxas. printing all known types and their default values is probably the right choice for an empty `--api-objects`.